kitsune/wiki/utils.py

```diff
@@ -290,3 +294,58 @@ def build_topics_data(request: HttpRequest, product: Product, topics: list[Topic
         topics_data.append(topic_data)
 
     return topics_data
+
+
+def clean_kb_visited(session, ttl=KB_VISITED_DEFAULT_TTL):
+    """
+    Remove expired slugs from the "kb-visited" dictionary within the given session
+    based on the given TTL.
+    """
+    if (not session) or ((kb_visited := session.get("kb-visited")) is None):
+        return None
+
+    now = time.time()
+
+    # Remove any slugs that have expired.
+    if expired_slugs := [slug for slug, ts in kb_visited.items() if (now - ts) > ttl]:
+        for slug in expired_slugs:
+            del kb_visited[slug]
+        session.modified = True
+
+    return kb_visited
```

Comment:
Probably it's cleaner if this method doesn't return anything and it just deletes (any) entries